### DISCUSSION TITLE
student verification approved but github pro does not applied.

### DISCUSSION THREAD

**Discussion by SihunLee-1016** [community/community#50551] [GitHub Education]
### Select Topic Area

Question

### Body

<img width="687" alt="image" src="https://user-images.githubusercontent.com/82656136/226505992-7e931da8-a782-4ca7-bebf-0ab60daa5501.png">

i verified in march 8, but github pro does not applied in my profile. its still beta. 
how can i solve this problem?

**Comment by AthulKkumar** [comment]
If you are are verified student just check out the student developer pack and search for Github in offers list and click the link throught it. The pro version will be automaticllay enable.
![Screenshot 2023-03-21 131744](https://user-images.githubusercontent.com/91550972/226545302-67744f45-7cf0-423f-9485-95f5cd36949f.png)

**Comment by sarahkjls** [comment]
i have the same problem. my student verification has been approved but github pro does not applied. I'm thankful for any help!

**Reply by Jossyboydgenius** [reply]
Mine is showing both beta and pro.
